README.md

```diff
@@ -1,7 +1,7 @@
 [![npm version](https://badge.fury.io/js/feathers-mongodb-fuzzy-search.svg)](https://badge.fury.io/js/feathers-mongodb-fuzzy-search) [![Build Status](https://travis-ci.org/arve0/feathers-mongodb-fuzzy-search.svg?branch=master)](https://travis-ci.org/arve0/feathers-mongodb-fuzzy-search)
 
 # feathers-mongodb-fuzzy-search
-Add fuzzy `$search` to mongodb `service.find` queries.
+Add fuzzy `$search` to mongodb `service.find` queries: full-text search on documents as well as pattern matching on individual fields.
```

Comment:
The purpose of this package is mainly full-text search (hence the name `fuzzy-search`).

Reply:
Actually the name is not so clear, fuzzy is also about approximate matching which is not what Mongo provide with `$search` (but more with `$regex`). Maybe `full-text-search` would be better in case you want to keep it as is.

Reply:
You are right about the confusing language, as this code does not do any stemming/approximations. Stemming might be a worthwhile addition, though not sure how you should handle strings with `'several words'`.